src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceUpdateUserTest.java

```diff
@@ -0,0 +1,114 @@
+package be.jidoka.jdk.keycloak.admin.service;
+
+import be.jidoka.jdk.keycloak.admin.IntegrationTest;
+import be.jidoka.jdk.keycloak.admin.domain.CreateUser;
+import be.jidoka.jdk.keycloak.admin.domain.UpdateUserBuilder;
+import be.jidoka.jdk.keycloak.admin.domain.UserAction;
+import org.junit.jupiter.api.Test;
+import org.keycloak.admin.client.resource.UsersResource;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Map;
+import java.util.Set;
+
+import static be.jidoka.jdk.keycloak.admin.domain.CreateUserFixture.bertenBoedhoe;
+import static be.jidoka.jdk.keycloak.admin.domain.CreateUserFixture.davitaOttervanger;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class KeycloakUserAdminServiceUpdateUserTest extends IntegrationTest {
+
+	@Autowired
+	private KeycloakUserAdminService keycloakUserAdminService;
+
+	@Autowired
+	private UsersResource usersResource;
+
+	@Test
+	void updatesAttributesAndRequiredActionsOnly() {
+		CreateUser createUser = bertenBoedhoe();
+		String userId = keycloakUserAdminService.createUser(createUser);
+
+		UpdateUserBuilder updateUser = UpdateUserBuilder.builder()
+				.userId(userId)
```

Comment:
The test claims to update something, but I don't see it here

Reply:
Renamed, extracted some basic assertions to a specific Assertion utility class.